pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/generators/protobuf/CodecParseMethodGenerator.java

```diff
@@ -109,38 +133,27 @@ static String generateParseMethod(final String modelClassName, final List<Field>
                                         throw new IOException("Cannot understand wire_type of " + wireType);
                                     }
                                     // It may be that the parser subclass doesn't know about this field
-                                    if (f == null) {
+                                    if ($prefixf == null) {
                                         if (strictMode) {
                                             // Since we are parsing is strict mode, this is an exceptional condition.
-                                            throw new UnknownFieldException(field);
+                                            throw new UnknownFieldException($prefixfield);
                                         } else {
                                             // We just need to read off the bytes for this field to skip it
                                             // and move on to the next one.
                                             skipField(input, ProtoConstants.get(wireType), $skipMaxSize);
                                         }
                                     } else {
-                                        throw new IOException("Bad tag [" + tag + "], field [" + field
+                                        throw new IOException("Bad tag [" + $prefixtag + "], field [" + $prefixfield
                                                 + "] wireType [" + wireType + "]");
                                     }
                                 }
                             }
                         }
-                        return new $modelClassName($fieldsList);
-                    } catch (final Exception anyException) {
-                        if (anyException instanceof ParseException parseException) {
-                            throw parseException;
-                        }
-                        throw new ParseException(anyException);
-                    }
-                }
                 """
-        .replace("$modelClassName",modelClassName)
-        .replace("$fieldDefs",fields.stream().map(field -> "    %s temp_%s = %s;".formatted(field.javaFieldType(),
```

Comment:
Seems like lots more complex replaces here than new code but I do not see them used in old code. Were they just old?

Reply:
This is just an odd way how git displays diffs. All the old replacement are still alive and well, and they reside where they always have - above at lines 80-90. I just refactored that above code and extracted the `parseLoop` generator into a dedicated method and here in this method we only need a few replacements. But the old replacements are still there - just a little above.

Again, this refactoring is done so that maps could generate their own `parseLoop` w/o me having to duplicate almost identical code generation code here.